dashboard/test/ui/features/applab/dataTab.feature

```diff
@@ -0,0 +1,13 @@
+@no_mobile
+@dashboard_db_access
```

Comment:
this test shouldn't need dashboard db access. please try removing this line, and then confirming that it passes on CircleCI or by running the test locally (I can help you with that).